src/components/_forms/WithdrawQueueForm.tsx

```
@@ -841,8 +843,10 @@ export const WithdrawQueueForm = ({
                   textAlign={"center"}
                   fontWeight={"bold"}
                 >
-                  When replacing a BoringQueue request, only the
-                  deadline is updated.
+                  {((useRouter().query.id as string) || _id) ===
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)